packages/@divvi/mobile/src/public/types.tsx

```diff
@@ -246,6 +246,10 @@ export interface PublicAppConfig<tabScreenConfigs extends TabScreenConfig[] = Ta
     }
     disableNfts?: boolean
     hideCashInTokenFilters?: boolean
+    // statsig feature gate defaults
+    showPositions?: boolean
+    showImportTokensFlow?: boolean
+    showSwapTokenFilters?: boolean
```

Comment:
I named these similar to feature gate names, but might be confusing since these default to true. Couple of other ideas
- Use opposite names (e.g., `hidePositions`) so they always default to false
- allow overriding all statsig gates (e.g., `statsigFeatureGateDefaults: { [key in StatsigFeatureGates]?: boolean}` (could be under statsig section) and link to defaults in documentation, which show defaults for all values

Reply:
i think opposite names are the best for builders since it's easy to understand what the config is doing? adding `statsig` in the name might be expositing too implementation detail unnecessarily....although having said this, the experimental object isn't really advised to be used by builders so i guess whichever option we choose is okay